subprojects/docs/src/docs/release/notes.md

```diff
@@ -220,6 +220,14 @@ Please note that this will not affect the runtime classpath that the `Test` task
 
 For more information about this feature, see the corresponding section in the [user guide](userguide/more_about_tasks.html#sec:custom_resource_normalization).
 
+### Convenience method for declaring Google repository
+
+Users of the Google repository `https://maven.google.com` can now use a shortcut method on `RepositoryHandler` to declare that dependencies should be used for dependency resolution. The following example demonstrate the repository declaration:
```

Comment:
rewording...
> A convenience method for the `[Google repository](https://developer.android.com/studio/build/dependencies.html#google-maven)` was added to `RepositoryHandler`. 
> You can now add Google's Maven repository to your build to resolve Android Support Library dependencies instead of downloading them from the Android SDK Manager. The following example demonstrates using the new shortcut repository declaration: